### PR TITLE
Ceds 4168 add link to confirmation page

### PR DESCRIPTION
**Pull request by magicaljellybeans**

### CODE REVIEW

File: app/controllers/storage/FlashKeys.scala [hunk 1]
```diff
@@ -18,4 +18,5 @@ package controllers.storage
 
 object FlashKeys {
```
Comment (udinobi) [inline app/controllers/storage/FlashKeys.scala:19]: As for External, what about moving `object Flashkeys` to FlashExtractor.scala as `object FlashExtractor` ?

**Review by udinobi — APPROVED** [review]

**Review by glidester — DISMISSED** [review]

File: app/controllers/storage/FlashExtractor.scala [hunk 2]
```diff
@@ -27,4 +27,11 @@ class FlashExtractor {
   def extractMovementType(request: Request[_]): Option[JourneyType] =
     extractValue(FlashKeys.MOVEMENT_TYPE, request).map(JourneyType.withName)
 
+  def extractUcr(request: Request[_]): Option[String] =
+    extractValue(FlashKeys.UCR, request)
+}
+
+object FlashKeys {
```
Comment (udinobi) [inline app/controllers/storage/FlashExtractor.scala]: I know, I'm a bit pedantic... still `object FlashKeys`? Not `object FlashExtractor`?

Comment (magicaljellybeans) [reply app/controllers/storage/FlashExtractor.scala]: My mistake, corrected now.

**Review by udinobi — APPROVED** [review]
Thank you :smiley:

**Pull request merged**